src/SiteInformation/FarmDates/EditDatesModal.js

```diff
@@ -0,0 +1,165 @@
+import {useSelector, useDispatch} from 'react-redux';
+import { setEditDatesModalData, setEditDatesModalOpen, setFarmDatesReloadNeeded } from '../../Store/actions';
+import React, { useState, useEffect } from 'react';
+import {Dialog, DialogContent, DialogTitle, DialogContentText, TextField, Button, Grid } from '@material-ui/core';
```

Comment:
there should also be a space here. enabling format on save should do all that for you though